libportal/camera.c

```diff
@@ -58,7 +59,8 @@ xdp_portal_is_camera_present (XdpPortal *portal)
       return FALSE;
     }
 
-  g_variant_get (ret, "(b)", &result);
+  g_variant_get (ret, "(v)", &prop);
+  result = g_variant_get_boolean (prop);
```

Comment:
You can remove the `result` variable now, and return `g_variant_get_boolean (prop)` directly

Reply:
done